gnd/src/main/java/com/google/android/gnd/ui/util/BitmapUtil.java

```diff
@@ -33,8 +39,26 @@
     this.context = context;
   }
 
-  /** Retrieves an image for the given url as a {@link Bitmap}. */
+  /**
+   * Retrieves an image for the given url as a {@link Bitmap}.
+   */
   public Bitmap fromUri(Uri url) throws IOException {
     return Media.getBitmap(context.getContentResolver(), url);
   }
+
+  public BitmapDescriptor bitmapDescriptorFromVector() {
+    Drawable vectorDrawable = ContextCompat.getDrawable(context, R.drawable.ic_endpoint);
```

Comment:
BitmapUtil is a generic utility, but `ic_endpoint` is specific to polygon drawing and thus doesn't belong hardcoded here. Please pass as an arg instead.

Reply:
updated